qualification/report/katdoc.sty

```diff
@@ -255,68 +249,3 @@
   \listofabbreviations
   \newpage{}
 }
-
-% Bibliography
-
-\renewenvironment{thebibliography}[1]
-{\list{\@biblabel{\@arabic\c@enumiv}}%
-  {\settowidth\labelwidth{\@biblabel{#1}}%
-    \leftmargin\labelwidth
-    \advance\leftmargin\labelsep
-    \itemindent 1.5em
-    \@openbib@code
-    \usecounter{enumiv}%
-    \let\p@enumiv\@empty
-    \renewcommand\theenumiv{\@arabic\c@enumiv}}%
-  \sloppy
-  \clubpenalty4000
-  \@clubpenalty \clubpenalty
-  \widowpenalty4000%
-  \sfcode`\.\@m}
-{\def\@noitemerr
-  {\@latex@warning{Empty `thebibliography' environment}}%
-  \endlist}
-
-\newcounter{countDocs}
-\setcounter{countDocs}{0}
-
-\newcommand{\addApplicableDocument}[5]{
-\item {[}AD\addtocounter{countDocs}{1} \arabic{countDocs}{]} \textbf{#2} \newline  #1, \textit{#3}, #4 \label{doc:#5}
-}
-
-\newcommand{\addReferenceDocument}[5]{
-\item {[}RD\addtocounter{countDocs}{1} \arabic{countDocs}{]} \textbf{#2} \newline  #1, \textit{#3}, #4 \label{doc:#5}
-}
-
-\newcommand{\references}[1]{
-  \subsection{Applicable and Referenced Documents}
-
-  \subsubsection{Applicable Documents}
-
-  The following documents are applicable to the extent stated herein.  In the event of conflict between the contents of the applicable documents and this document, the applicable documents shall take precedence.
-
-  \ifdefined\applicableDocs
-    \begin{enumerate}
-      \applicableDocs
-    \end{enumerate}
-  \else
-    \begin{enumerate}
-      \item \emph{No applicable documents}
-    \end{enumerate}
-  \fi
-
-  \subsubsection{Reference Documents}
-
-  The following documents are referenced in this document.  In the event of conflict between the contents of the referenced documents and this document, this document shall take precedence.
-
-  \ifdefined\referenceDocs
-    \begin{enumerate}
-      \referenceDocs
-    \end{enumerate}
-  \else
-    \begin{enumerate}
-      \item \emph{No reference documents}
-    \end{enumerate}
-  \fi
```

Comment:
Ah, good call.

Reply:
Yeah, in the old katdoc.sty there were hacky ways to get the bibliography in. We've changed that because we didn't like it. This was just the clutter that has been cut.